web-lib-funcs.pl

```diff
@@ -5622,6 +5625,30 @@ sub get_module_info
 return %rv;
 }
 
+=head2 get_module_overrides($mod, \data)
+
+Checks for module specific overrides if exist in
+module.overrides file and executes defined subs
+
+=cut
+sub get_module_overrides
+{
+my ($mod, $data) = @_;
+return if (!$mod);
+
+my $mdir = &module_root_directory($mod);
+
+# Call module specific overrides
+my $call = 'module_overrides';
+if (-r "$mdir/$call.pl") {
+	eval {
+		local $main::error_must_die = 1;
+		&foreign_require($mod, "$call.pl");
+		&foreign_call($mod, $call, \%{$data});
```

Comment:
Why \%{$data} rather than just $data ?

Reply:
It wouldn't alter the data, if not passed as reference?

Reply:
Yes, but isn't the whole idea to alter the data?

Reply:
🙂 Yes, the whole point is to alter the data and passing it as `$data` the second time wouldn't work. No?

Reply:
Since $data is a hash reference, you can alter the contents by just passing it as $data

Reply:
Actually you're right! Fixed! Thanks for the review.